src/actions/breakpoints.js

```diff
@@ -191,6 +191,34 @@ export function toggleAllBreakpoints(shouldDisableBreakpoints: boolean) {
   };
 }
 
+export function remapBreakpoints(sourceId: string) {
```

Comment:
what do you think of moving `remapBreakpoints` to `actions/breakpoints/remapBreakpoints`?